front/cypress/integration/admin/projects/use_content_builder.ts

```diff
@@ -130,7 +130,6 @@ describe('Admin: add project and edit description', () => {
       position: 'inside',
     });
     cy.get('#e2e-image').should('exist');
-    cy.get('#e2e-image').click();
     cy.get('#e2e-delete-button').click();
```

Comment:
I realized that we didn't need this click anyways, as the Settings panel is already open when the component is added.